README.md

```diff
@@ -17,16 +23,17 @@ attendees a strong foundation in the fundamentals of R, and to teach best
 practices for scientific computing: breaking down analyses into modular units,
 task automation, and encapsulation.
 
-Note that this workshop focuses on the fundamentals of the programming
-language R, and not on statistical analysis.
-
-The lesson contains more material than can be taught in a day.  The [instructor notes page]({{ page.root }}/guide) has some suggested lesson plans suitable for a one or half day workshop.
+This lesson has been expanded to incorporate additional Software Carpentry content 
+on the use of Git for Version Control, navigating files and directories in a terminal, 
+SQL and new content on the creation and validation of regression models.
 
 A variety of third party packages are used throughout this workshop. These
 are not necessarily the best, nor are they comprehensive, but they are
 packages we find useful, and have been chosen primarily for their
 usability.
 
+## R for Reproducible Scientific Analysis
+
 Current Maintainers:
```

Comment:
May want to remove this or add our own names and then a different section to acknowledge the maintainers of the original material?